dumpling/export/status_test.go

```diff
@@ -47,6 +48,8 @@ func TestSpeedRecorder(t *testing.T) {
 	for _, tc := range testCases {
 		time.Sleep(time.Duration(tc.spentTime) * time.Second)
 		recentSpeed := speedRecorder.GetSpeed(tc.finished)
-		require.Equal(t, tc.expected, recentSpeed)
+		if math.Abs(tc.expected-recentSpeed) > 0.1 {
+			require.FailNow(t, "speed unexpected")
```

Comment:
print unexpected speed?

Reply:
no, a check error, fixed https://github.com/pingcap/tidb/pull/38625/commits/22c293c1728b4563a9479e89af3558a7cb98fd00

Reply:
I mean print `recentSpeed` and `tc.expected` here for debugging.